server/application-server/src/main/java/de/tum/in/www1/hephaestus/chat/session/SessionRepository.java

```diff
@@ -0,0 +1,14 @@
+package de.tum.in.www1.hephaestus.chat.session;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+import java.util.List;
+
+@Repository
+public interface SessionRepository extends JpaRepository<Session, Long> {
+
+    Optional<List<Session>> findByUserLogin(@Param("login") String login);
```

Comment:
If you don't specify `@Query`, you shouldn't need `@Param` either.

Also lists don't need to be optional as you would just get an empty list in case the database returns an empty result set.